cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/strategy/PostponeUrpReplicaMovementStrategy.java

```diff
@@ -17,11 +18,36 @@ public class PostponeUrpReplicaMovementStrategy extends AbstractReplicaMovementS
 
   @Override
   public Comparator<ExecutionTask> taskComparator(StrategyOptions strategyOptions) {
-    return (task1, task2) -> isPartitionUnderReplicated(strategyOptions.cluster(), task1.proposal().topicPartition())
-                             ? (isPartitionUnderReplicated(strategyOptions.cluster(), task2.proposal().topicPartition()) ? PRIORITIZE_NONE
-                                                                                                                         : PRIORITIZE_TASK_2)
-                             : (isPartitionUnderReplicated(strategyOptions.cluster(), task2.proposal().topicPartition()) ? PRIORITIZE_TASK_1
-                                                                                                                         : PRIORITIZE_NONE);
+    return (task1, task2) -> {
```

Comment:
It would be great to have some unit tests for this one.

Reply:
Done